src/components/nav.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { Link } from "gatsby";
+import { Menu } from "feather-icons-react";
```

Comment:
this dependency is missing, add it with `npm i --save feather-icons-react` and commit the changes on package.json and package-lock.json